.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.1
```

Comment:
Why 2.3.1?

Reply:
Hermod requires an older version of bundler 1.6.1. I started with using 2.6.3 but it gave bundler error, so switched to 2.3.1. 

Available versions are:
https://github.com/fac/puppet-config/blob/production/role_modules/jenkins_ci_role/manifests/rubies.pp

Reply:
my bad. changed it to 2.6